src/Result.ts

```diff
@@ -1,40 +1,11 @@
 import type {Result as AxeResult, NodeResult} from 'axe-core';
 import indent from 'indent-string';
-import type {Page} from 'playwright';
 import dedent from 'ts-dedent';
-import type ProcessedStory from './ProcessedStory';
-import {analyze} from './browser/AxePage';
-
-/**
- * These rules aren't useful/helpful in the context of Storybook stories, and we disable them when
- * running Axe.
- */
-const defaultDisabledRules = [
-  'bypass',
-  'landmark-one-main',
-  'page-has-heading-one',
-  'region',
-];
```

Comment:
Moved the default disabled rules into the AxePage file, so that Axe stuff is more centralized there.